builtin/providers/aws/resource_aws_subnet.go

```diff
@@ -51,15 +51,15 @@ func resourceAwsSubnet() *schema.Resource {
 }
 
 func resourceAwsSubnetCreate(d *schema.ResourceData, meta interface{}) error {
-	ec2conn := meta.(*AWSClient).ec2conn
+	conn := meta.(*AWSClient).ec2SDKconn
```

Comment:
I'd like to consolidate this connection var across AWS resources as simply `conn`, instead of `ec2conn` and `r53` conn etc. 

Open for discussion.